language/Portuguese_Brazil/strings.po

```diff
@@ -200,35 +200,35 @@ msgstr "Adicionar filtro"
 
 #: src/frontend/mame/ui/utils.cpp:950
 msgid "Select category:"
-msgstr ""
+msgstr "Selecione a categoria"
 
 #: src/frontend/mame/ui/utils.cpp:981
 msgid "[no category INI files]"
-msgstr ""
+msgstr "[arquivos INI sem categorias]"
 
 #: src/frontend/mame/ui/utils.cpp:989
 msgid "[no groups in INI file]"
-msgstr ""
+msgstr "[sem grupos no arquivo INI]"
 
 #: src/frontend/mame/ui/utils.cpp:1023
 msgid "No category INI files found"
-msgstr ""
+msgstr "arquivos INI sem categoria encontrada"
```

Comment:
Google Translate suggests: "Nenhum arquivo INI de categoria foi encontrado" for "No category INI files were found" (yes, I know I added the "were").